### DISCUSSION TITLE
Operator for Artemis 2.31.2

### DISCUSSION THREAD

**Discussion by Mohanavalli-A** [arkmq-org/arkmq-org-broker-operator#744] [General] [answered]
Hi Team,

We are looing to upgrade the Artemis broker to 2.31.2 to fix the below CVE.

https://activemq.apache.org/news/cve-2023-46604

Operator supporting v2.31.2 is not released yet. When is this expected to be released?

Thanks in advance!

**Comment by brusdev** [comment, marked as answer]
ArtemisCloud ActiveMQ Artemis Operator 1.0.16 thats support Apache ActiveMQ Artemis 2.31.2 has been released https://github.com/artemiscloud/activemq-artemis-operator/releases/tag/v1.0.16
